src/secp256k1.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable */
```

Comment:
What was ESLint getting angry about?

Reply:
Tons of stuff; we can remove the line and run the linter to see. It might be worth going through and adding types in this file but that might be large enough for its own PR.